src/components/Navbar.jsx

```diff
@@ -4,12 +4,62 @@ import './styles/Navbar.css';
 export default function Navbar() {
   return (
     <div className="bookstore-nav">
-      <div className="bookstore-nav-left">
-        <h1 className="book-head">Bookstore CMS</h1>
+      <div className="nav-left">
+        <h1 className="transparent">Bookstore CMS</h1>
         <NavLink to="/">Book</NavLink>
         <NavLink to="categories">Category</NavLink>
       </div>
```

Comment:
I strongly recommend reviewing the font-families you are using because they are not the proposed fonts in the template. The font for the navigation is Montserrat and the font for the book item is Roboto-slab except for the completed text which is Montserrat. Consider addressing this issue in order to have a design that tallies the template.